src/views/open-api-docs-view/components/open-api-sidebar-contents/index.tsx

```diff
@@ -22,9 +22,11 @@ import s from './open-api-sidebar-contents.module.css'
 export function OpenApiSidebarContents({
 	navItems,
 	navResourceItems,
+	showFilterInput = true,
 }: {
 	navItems: OpenApiNavItem[]
 	navResourceItems: OpenApiNavItem[]
+	showFilterInput?: boolean
```

Comment:
Toggles visibility of this component. Not part of the install page updates.
<img width="1087" alt="Screenshot 2023-11-29 at 7 39 09 PM" src="https://github.com/hashicorp/dev-portal/assets/55773810/d0d7d379-ab44-4114-834d-f94dbc19cd1b">

Reply:
The install page sidebar now uses the `<OpenApiSidebarContents>` component (implementation [here](<OpenApiSidebarContents)]. Previously, this component was only used on the API docs page (as in the screenshot above). The mocks for the install page did not include the search input, so I added a boolean prop that will allow `showFilterInput=False` to hide the search input on the product install page. Nothing has changed on the API docs page. Does that answer your question?